fe1-web/src/features/evoting/hooks/EvotingHooks.ts

```diff
@@ -0,0 +1,38 @@
+import FeatureContext from 'core/contexts/FeatureContext';
+import { useContext } from 'react';
+import { EVOTING_FEATURE_IDENTIFIER } from '../index';
+import { EvotingReactContext } from '../objects';
+
+export namespace EvotingHooks {
+  export const useEvotingContext = (): EvotingReactContext => {
+    const featureContext = useContext(FeatureContext);
+    // assert that the evoting context exists
+    if (!(EVOTING_FEATURE_IDENTIFIER in featureContext)) {
+      throw new Error('Evoting context could not be found!');
+    }
+    return featureContext[EVOTING_FEATURE_IDENTIFIER] as EvotingReactContext;
+  };
+  /**
+   * Gets the current lao id
+   * @returns The current lao id
+   */
+  export const useCurrentLaoId = () => {
+    return useEvotingContext().getCurrentLaoId();
+  };
```

Comment:
This might not behave the way you want, we probably need to discuss this in-person.
There are subtleties to the use of `make`Selectors, useSelectors and memoization of those.
The same applies to the other hooks here.